apps/dotcom/src/components/LocalEditor.tsx

```diff
@@ -106,6 +105,7 @@ export function LocalEditor() {
 				onUiEvent={handleUiEvent}
 				components={components}
 				inferDarkMode
+				getUrlInfoForBookmark={getUrlInfoForBookmark}
```

Comment:
can't there be a more generic entrypoint (like component overrides)? feels very one-off for a particular shape.